src/deployment/flavors.md

```diff
@@ -3,41 +3,171 @@ title: Creating flavors for Flutter
+Flavors (known as _build configurations_ in iOS), allow you (the developer) to 
+create separate environments for your app using the same code base. 
+For example, you might have one flavor for your full-fledged production app, 
+another as a limited "free" app, another for testing experimental features, and so on. 
+
+Let's say you want to make both free and paid versions of your Flutter app. 
+You can use flavors to set up both app versions 
+without writing two separate apps. 
+The free version of the app has basic functionality and ads (for example). 
+In contrast, the paid version has basic app functionality, extra features, 
+different styles for paid users, and no ads. 
+
+Another example where you might use flavors is for feature development. 
+If you’ve built a new feature and want to try it out, 
+you could set up a flavor to test out the feature. 
+Your production code remains unaffected 
+until you are ready to deploy your new feature.
+
+Flavors function by letting you define compile time configurations 
+and set parameters that are read at runtime depending on your app needs.
+
+This document guides you through setting up Flutter flavors for iOS and Android. 
+
+## Environment set up
+Prerequisites:
+* Xcode installed
+* An existing project 
+
+To set up flavors in iOS, you’ll define build configurations in Xcode. 
+
+## Creating flavors in iOS
+1. Open your project in Xcode.
+
+2. Select **Runner** > **New Scheme** from the menu to add a new Scheme.  
+* A scheme describes how Xcode runs different actions. 
+  For the purposes of this guide, the example _flavor_ and _scheme_ are 
+  named `development`. 
+  The build configurations in the development scheme 
+  have the `-development` suffix. 
+
+3. Duplicate the build configurations to differentiate between the 
+default configurations that are already available and the new configurations 
+for the `development` scheme. 
+* Under the **Info** tab at the end of the 
+**Configurations** dropdown list, click the plus button and duplicate 
+each configuration name (Debug, Release, and Profile). 
+Duplicate the listed production configurations, once for each environment. 
+
+![Step 3 Xcode image](/assets/images/docs/flavors/step3-ios-build-config.png){:width="100%"}
+{{site.alert.note}}
+  Your configurations should be based on your **Debug.xconfig** or **Release.xcconfig**
+  file, not the **Pods-Runner.xcconfigs**. You can check this by expanding the configuration names. 
+{{site.alert.end}}
+
+&nbsp;4. To match the development flavor, add `-development` 
+at the end of each new configuration name. 
+
+&nbsp;5. Change the `development` scheme to match the development 
+build configurations already created.
+* In the **Runner** project, click **Manage Schemes…** and a pop up window opens. 
+* Double click the development scheme. In the next step 
+(as shown in the screenshot), you’ll modify each scheme 
+to match its dev build configuration:
+
```

Comment:
The bullet points under the numbered points in this section don't match up. I'm guessing it has something to do with how 4 and 5 weren't rendering properly. So if this is the best solution for that issue, it's good enough! Just bringing it to your attention in case you didn't see